bin/run-tests.sh

```diff
@@ -12,6 +12,8 @@
 # ./bin/run-tests.sh
 
 exit_code=0
+test_counter=0
```

Comment:
Maybe change this to:

```suggestion
test_success_counter=0
```

Note: you'd also need to update this where it is used